src/src/ExpressionChecker.java

```diff
@@ -1,6 +1,7 @@
-class ExpressionChecker {
+public class ExpressionChecker {
+	static final float checkFloat = 1E-4;
```

Comment:
add access modifier and read about constant names
https://mate-academy.github.io/style-guides/java/java.html#s5-naming

Reply:
Access modifier added and change the name of constant.